scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -536,6 +536,7 @@ def _update_instance_info(self, harness_state, handler_time, flash_error):
             self.instance.status = harness_state
             if harness_state == "failed":
                 self.instance.reason = "Failed"
+            self.instance.add_missing_case_status("blocked", harness_state)
```

Comment:
```suggestion
            self.instance.add_missing_case_status("blocked", harness_state)
```
`if` to set reason is not needed now with  add_missing_case_status() call, isn't it ?